fury-on-gke/README.md

```diff
@@ -545,183 +623,134 @@ Navigate to <http://grafana.fury.info> or click the Grafana icon from Forecastle
 Fury provides pre-configured dashboards to visualize the state of the cluster. Examine an example dashboard:
 
 1. Click on the search icon on the left sidebar.
-
 2. Write `pods` and click enter.
-
 3. Select the `Kubernetes/Pods` dashboard.
 
 This is what you should see:
 
 ![Grafana][grafana-screenshot]
 
-## Step 5 (optional) - Deploy additional modules
-
-We now install other modules:
-
-- dr
-- opa
+## Step 5 (optional) - Advanced Distribution usage
 
-To deploy Velero as a Disaster Recovery solution, we need to have credentials to interact with GKE volumes.
+### (optional) Create a backup with Velero
 
-1. Let's add a module at the bottom of `Furyfile.yml`:
-
-```yaml
-versions:
-  ...
-  dr: v1.10.0
-  opa: v1.7.3
-
-bases:
-  ...
-  - name: dr
-  - name: opa
-
-modules:
-- name: dr
-```
-
-2. And download the new vendor folders:
+1. Create a backup with the `velero` command-line utility:
 
 ```bash
-furyctl vendor -H
+velero backup create --from-schedule manifests test -n kube-system
 ```
 
-3. Create the resources using Terraform:
+2. Check the backup status:
 
 ```bash
-cd /demo/terraform/demo-fury
-# Update the provided `main.tf` file with your configuration.
-terraform init
-terraform plan -out terraform.plan
-terraform apply terraform.plan
-
-# Output the resources to yaml files, so we can use them in kustomize
-terraform output -raw velero_backup_storage_location > ../../manifests/resources/velero-backup-storage-location.yml
-terraform output -raw velero_volume_snapshot_location > ../../manifests/resources/velero-volume-snapshot-location.yml
-terraform output -raw velero_cloud_credentials > ../../manifests/resources/velero-cloud-credentials.yml
-
+velero backup get -n kube-system
 ```
 
-4. Add the following lines to `kustomization.yaml`:
+### (optional) Enforce a Policy with OPA Gatekeeper
```

Comment:
you can copy this section from my open PR [#27](https://github.com/sighupio/fury-getting-started/issues/27)